scripts/create-package.js

```diff
@@ -1,34 +1,54 @@
-// Dependencies
 const mustache = require('mustache');
 const argv = require('yargs').argv;
 const fs = require('fs');
 const exec = require('./exec-sync');
+const readConfig = require('./read-config');
```

Comment:
@kkjeer Looks like you wrote the create-package script--would you mind taking a look at these changes?